spring-integration-redis/src/main/java/org/springframework/integration/redis/util/RedisUtils.java

```diff
@@ -36,9 +37,11 @@
 
 	private static final String VERSION_PROPERTY = "redis_version";
 
+	@SuppressWarnings("serial")
 	private static final Map<RedisOperations<?, ?>, Boolean> unlinkAvailable =
 			new LinkedHashMap<RedisOperations<?, ?>, Boolean>() {
 
+				@SuppressWarnings("rawtypes")
```

Comment:
Can't we fix it with an explicit generic args in the `Map.Entry` definition instead?

But thank you for catching this one!